stats.go

```diff
@@ -39,7 +40,7 @@ type DiscardStats struct{}
 
 func (d *DiscardStats) LogStartup()                                   {}
 func (d *DiscardStats) LogClientCount(int64)                          {}
-func (d *DiscardStats) LogGoroutines(int)                          {}
+func (d *DiscardStats) LogGoroutines(int)                             {}
 func (d *DiscardStats) LogCommand(from, cmdType string)               {}
 func (d *DiscardStats) LogPageMessage()                               {}
 func (d *DiscardStats) LogUserMessage()                               {}
```

Comment:
Absolutely. I apologize at first it was only the initial formatting. Just kept adding stuff while I went along. I will clean everything and make sure it's more clear of what changes where made. 

Thanks!